Userland/Utilities/mkdir.cpp

```diff
@@ -41,14 +43,24 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
 
     bool has_errors = false;
 
+    auto create_directory = [&](StringView path, mode_t mode) {
+        auto maybe_error = Core::System::mkdir(path, mode);
+        if (maybe_error.is_error()) {
+            warnln("mkdir: {}", strerror(maybe_error.error().code()));
+            has_errors = true;
+            return false;
+        }
+
+        if (verbose)
+            outln("Created directory '{}'", path);
```

Comment:
It's not like I prefer it, but that's not consistent with `rmdir`  which add a `rmdir: ` prefix